components/builder-depot/src/server.rs

```diff
@@ -326,7 +326,12 @@ fn remove_file(filename: &PathBuf) -> Result<bool> {
 }
 
 fn upload_origin_key(req: &mut Request) -> IronResult<Response> {
-    let depot = req.get::<persistent::Read<Depot>>().unwrap();
+    // JB TODO: the following two lines are a temporary hack for the channels work, because we're
+    // using an in-memory HashMap to store channel data instead of a proper datastore. These should
+    // be removed (along with all of the other lines like this in this file) and persistent::Read
+    // should be put back once the depot is using PG for storage.
+    let lock = req.get::<persistent::State<Depot>>().unwrap();
```

Comment:
@raskchanky IMO we shouldn't handle the error, but we probably want to use `expect()` instead of `unwrap()` like we had before. This is the equivalent of a "debug_assert" which let's the developer know they made a mistake before it goes out to production. There shouldn't be a case where the plugin isn't loaded and, if it is, then the developer made a mistake.